docs/source/Appendix_B.md

```diff
@@ -409,7 +409,14 @@ under an appropriate unit class).
 
 #### SCHEMA_CHARACTER_INVALID
 
-**a.**  The specification contains an invalid character for the section in which it appears.  
+**a.** A schema node contains non `name` characters.  
+**b.** A unit class name contains non `name` characters.  
+**c.** A unit contains characters non `name` characters.  
+**d.** A value class name contains non `name` characters.  
+**e.** The prologue or epilogue contain characters other than `text` or newline.  
+**f.** A unit name contains a blank.  

```

Comment:
this should probably be rolled into c above, and c should note it's unit name(rather than just unit) as well

Reply:
Combined c, d, and f.  Added a note.